tests/codemods/test_use_set_literal.py

```diff
@@ -0,0 +1,43 @@
+from core_codemods.use_set_literal import UseSetLiteral
+from tests.codemods.base_codemod_test import BaseCodemodTest
+
+
+class TestUseSetLiteral(BaseCodemodTest):
+    codemod = UseSetLiteral
+
+    def test_simple(self, tmpdir):
+        original_code = """
+        x = set([1, 2, 3])
+        """
+        expected_code = """
+        x = {1, 2, 3}
+        """
+        self.run_and_assert(tmpdir, original_code, expected_code)
+
```

Comment:
worth adding `assert len(self.file_context.codemod_changes) == 1` (or 0) to each test case